api/src/main/java/com/github/eyefloaters/console/api/service/TopicService.java

```diff
@@ -307,7 +313,7 @@ List<OffsetSpec> getRequestOffsetSpecs(String offsetSpec) {
         return specs;
     }
 
-    List<org.apache.kafka.common.TopicPartition> getTopicPartitions(Map<Uuid, Either<Topic, Throwable>> topics, Map<String, Uuid> topicIds) {
+    Map<TopicPartition, List<Integer>> getTopicPartitionNodes(Map<Uuid, Either<Topic, Throwable>> topics, Map<String, Uuid> topicIds) {
```

Comment:
Would `getTopicPartitionReplicas` be a better name?

Does it need the `get` prefix, if it's not a JavaBean getter?

Reply:
Renamed `getTopicPartitionNodes` to `topicPartitionReplicas`